src/pages/availability-panel.tsx

```diff
@@ -17,6 +18,18 @@ const AvailabilityPanel: FC = () => {
   const [isMemberLoading, setIsMemberLoading] = useState<boolean>(true);
   const [refreshData, setRefreshData] = useState<boolean>(false);
 
+  const getAndSetIdleUsers = async () => {
+    try {
+      const [idleUserNames, error] = await fetchIdleUsers();
+      setIdleMembersList(idleUserNames);
+      setError(error);
+    } catch (err) {
+      setError(true);
+    } finally {
+      setIsMemberLoading(false);
+    }
```

Comment:
You can also remove this try-catch block as in case of an error `fetchIdleUsers` function is returning an empty array and the error is not being thrown from there. 

Here is an example [link](https://www.typescriptlang.org/play?#code/MYewdgzgLgBMCu0QFsBiBTKwAWMC8MY6A7jAAoBOKAlhOgBT0XoQgA2AbugDQzMBW6YFACU+AHwwA3gCgY8uOGh8AhmAAm+GAFkVUbADoAZmxAgK9XfoMU16lPTEAqGAEYADCLkLqRmEzsYSQBWEVkFCL4Wdi56AG0AchUAI2B1BN4E9CMAc2wEgF0vCIBfGW95ASEoegAiVBVqNnRNKBAYUxVNdT0VWq8SrxlQSFgjTBwASXV0FTYAVToKCC0VCABPMGB-EQAuSho6AB5oCmowHLiCyTxJcIUoCnX7yMVRmB6oFVXiRtgEJBoCa4NYwU7nS4FCqRZhQeAUMAfXrQkrAPQ4ejoMLQiKw+GIq4omRlYZKWCddQAEV6qw2Wx2EmkMGhI2Un2+BBUv2oY2B01mCyWEEcLKU7HQBlMOXoACkAMoAeQAcgZwRdfOt6OzeGB4Gw2LwAEwiAblCnUr6OIA) you can test this behaviour on.

Reply:
Removed the redundant try-catch block inside `getAndSetIdleUsers` function